### PR TITLE
Update of Ukrainian Translation

### PR DESCRIPTION
**Pull request by ZaHaDumGoth**

### CODE REVIEW

File: files/lang/uk.po [hunk 1]
```diff
@@ -5245,7 +5226,7 @@ msgstr ""
 "на одній карті."
 
 msgid "fheroes2 Resurrection Team presents"
-msgstr ""
+msgstr "fheroes2 Resurrection Team представляє"
```
Comment (ihhub) [inline files/lang/uk.po:5229]: I think the word **team** must be translated as well.

**Review by ihhub — CHANGES_REQUESTED** [review]
Hi @ZaHaDumGoth , I left comment here. Could you please take a look at it once you have time?

File: files/lang/uk.po (hunk 1 above)
Comment (ZaHaDumGoth) [reply files/lang/uk.po:5229]: That is a complete name or only "fheroes2 Resurrection" ?
Or it means team as personnel/people related to project with name "fheroes2 Resurrection" ?

Comment (ihhub) [reply files/lang/uk.po:5229]: `fheroes2 Resurrection` is a name of a team of people who are making this project. Like "Piano movers" team. Therefore, `fheroes2 Resurrection` is the complete name.

Comment (ZaHaDumGoth) [reply files/lang/uk.po:5229]: Clear. Thanks. I fixed it minute ago and did Commit.

Comment (ihhub) [reply files/lang/uk.po:5229]: Emm... I don't see an extra commit. Are you sure that you committed to the same branch as this pull request has?

Comment (ZaHaDumGoth) [reply files/lang/uk.po:5229]: I presse Save in PoEdit, Pressed Commit in GitHub DSKT and Pull Origin, but some conflik appeared
Screenshot_1

Comment (ihhub) [reply files/lang/uk.po:5229]: Let's do this way: paste the update translation for this string here. I will replace it.

Comment (ZaHaDumGoth) [reply files/lang/uk.po:5229]: Ok, here is translated row - Команда fheroes2 Resurrection представляє

**Review by ihhub — APPROVED** [review]

File: files/lang/uk.po [hunk 2]
```diff
@@ -4047,11 +4030,11 @@ msgid "Quit out of the map editor."
 msgstr "Вийти з редактора мап."
 
 msgid "Input %{object} text"
-msgstr ""
+msgstr "Ввести %{object} текст"
 
 #, fuzzy
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:4035]: Please clarify where is this text in game.

File: files/lang/uk.po [hunk 3]
```diff
@@ -4683,25 +4666,25 @@ msgid "Special Thanks to"
...
-msgstr ""
+msgstr "та багато-багато інших дописувачів і підтримувачів!"
 
 msgid "Support us at"
-msgstr ""
+msgstr "Підтримайте нас нас "
 
 msgid "local-donation-platform|https://www.patreon.com/fheroes2"
-msgstr ""
+msgstr "Місцева платформа донатів|https://www.patreon.com/fheroes2"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:4675]: Fixed that.

File: files/lang/uk.po [hunk 4]
```diff
@@ -4683,25 +4666,25 @@ msgid "Special Thanks to"
...
 
 msgid "Connect with us at"
-msgstr ""
+msgstr "Зв'яжіться з нами на"
 
 msgid "local-social-network|https://www.facebook.com/groups/fheroes2"
-msgstr ""
+msgstr "локальна соціальна мережа|https://www.facebook.com/groups/fheroes2"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:4681]: Fixed.

File: files/lang/uk.po [hunk 5]
```diff
@@ -9019,22 +9043,22 @@ msgid "Frozen Lake"
...
-msgstr ""
+msgstr "випадкова раса|істоти 4-го рівня"
 
 msgid "randomRace|level 5 creatures"
-msgstr ""
+msgstr "випадкова раса|істоти 5-го рівня"
 
 msgid "randomRace|level 6 creatures"
-msgstr ""
+msgstr "випадкова раса|істоти 6-го рівня"
```
[the remark this answers is not in the document]
Comment (ZaHaDumGoth) [reply files/lang/uk.po:9061]: Fixed all.

**Comment by ihhub** [comment]
@ZaHaDumGoth , a bunch of thanks for the update!

**Pull request merged**